packages/thumbprint-react/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Fixed
+
+-   [Patch] Fix fatal error being thrown when passing `null` to the `value` prop of `Calendar`.
```

Comment:
Was this actually fixed?

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/916038/99679015-75119d80-2a49-11eb-8b38-722a3d4d3abc.png">

Maybe we should add a regression test?